File: Kattis/QALY/qaly.rb
n = gets.to_i;
if n >= 1 and n <= 100 then
	arr = []
	n.times do
		x, y = gets.chomp.split.map(&:to_f);
		if x > 0 and x <= 1 and y > 0 and y <= 100 then
			arr.push(x * y);
		end
	end
end

print arr.sum

File: Kattis/QALY/qaly.py
n = int(input())
if n >= 1 and n <= 100:
	arr = []
	for i in range(n):
		lst = list(input().split())
		x, y = float(lst[0]), float(lst[1])
		if x > 0 and x <= 1 and y > 0 and y <= 100:
			arr.append(x * y)

	print(sum(arr))